admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -477,6 +479,16 @@ class AdminModuleController {
     let newValue;
     let defaultMax;
 
+    const paramsUrl = (new URL(document.location)).searchParams;
+    const findModule = paramsUrl.get('find_module');
```

Comment:
I suggest using "search" instead of "find_module", because it's more coherent with the name of the field.

The field should also not be a tag list, but that's a whole different story, and is outside the scope of this change 😅

Reply:
I apply your suggest @prestonBot